README.md

```diff
@@ -1,15 +1,24 @@
 # **Cholesky Factorization**
 
 ## **Index**
-1. [Introduction](#introduction)
-    - [Positive semidefinite matrices](#positive-semidefinite-matrices)
-2. [The Cholesky algorithm](#the-cholesky-algorithm)
-    - [The Cholesky–Banachiewicz and Cholesky–Crout algorithms](#the-cholesky–banachiewicz-and-cholesky–crout-algorithms)
-    - [The Computation by Diagonal Algorithm](#the-computation-by-diagonal-algorithm)
-3. [Implementation Info and comparison with other methods](#implementation-info-and-comparison-with-other-methods)
-4. [Results](#results)
-5. [Installation and env preparation](#installation-and-virtual-environment-preparation)
-6. [Execution Guide](#execution-guide)
+- [**Cholesky Factorization**](#cholesky-factorization)
+  - [**Index**](#index)
+  - [**Introduction**](#introduction)
+  - [**Positive semidefinite matrices**](#positive-semidefinite-matrices)
+- [**The Cholesky algorithm**](#the-cholesky-algorithm)
+  - [**The Cholesky–Banachiewicz and Cholesky–Crout algorithms**](#the-choleskybanachiewicz-and-choleskycrout-algorithms)
+  - [**The Diagonal by Diagonal Computation Algorithm**](#the-diagonal-by-diagonal-computation-algorithm)
+  - [**Implementation Info and comparison with other methods**](#implementation-info-and-comparison-with-other-methods)
+    - [**Directory content explaination**](#directory-content-explaination)
+  - [**Results**](#results)
+    - [**Cholesky Factorization (no JIT)**](#cholesky-factorization-no-jit)
```

Comment:
non credo ci sia bisogno di indicizzare ogni tabella (confonde molto l'indice dalle altre cose importanti)

Reply:
Risolto in 3892367